src/types.ts

```diff
@@ -75,20 +76,3 @@ export interface WebhookEventHandlerError<TTransformed = unknown>
     ? EmitterWebhookEvent
     : EmitterWebhookEvent & TTransformed;
 }
-
```

Comment:
The types were fixed in version 3.0.1 and these are out of date versus v5 of `aggregate-error`